### ISSUE TITLE
ARIA hide SVG icon and <canvas> elements presented as "image"

### ISSUE THREAD

**Issue by joaopalmeiro** [evilmartians/oklch-picker#202]
Hi! 👋 

While navigating sequentially through the app, I noticed more images that are read by VoiceOver (in Chrome, in Safari it's okay):

- Down SVG icon on the menu button to change the format: https://github.com/evilmartians/oklch-picker/blob/9b9adbf86176e5c0d95645ff582de66f08714164/view/field/mixin.pug#L14 + https://github.com/evilmartians/oklch-picker/blob/9b9adbf86176e5c0d95645ff582de66f08714164/view/field/down.svg
- `<canvas>` elements for the sliders: https://github.com/evilmartians/oklch-picker/blob/9b9adbf86176e5c0d95645ff582de66f08714164/view/range/mixin.pug#L4-L10 (maybe `aria-hidden="true"` should be applied to the color slider container instead?)

I think it would be useful to add `aria-hidden="true"`. What do you think?
